command/operator_debug.go

```diff
@@ -251,9 +251,27 @@ func (c *OperatorDebugCommand) Run(args []string) int {
 		}
 	}
 
-	// Resolve server prefixes
-	for _, id := range argNodes(serverIDs) {
-		c.serverIDs = append(c.serverIDs, id)
+	// Resolve servers
+	members, err := client.Agent().Members()
+	c.writeJSON("version", "members.json", members, err)
+	// We always write the error to the file, but don't range if no members found
+	if members != nil { // members
```

Comment:
do we need this check here? if members is nil or returns an error, do we still want to set `c.serverIDs` equal to the ones that were passed in?

I think the members nil check could get moved down to where its used
```go
		if serverIDs == "all" && members != nil {
			// Special case to capture from all servers
			for _, member := range members.Members {
				c.serverIDs = append(c.serverIDs, member.Name)
			}
		} else {
			for _, id := range argNodes(serverIDs) {
				c.serverIDs = append(c.serverIDs, id)
			}
		}
```

Reply:
This should be fine, but a couple of the unit tests have been failing since the change.  Feels like an unrelated race condition crept in somewhere.  I think waiting for leader election should handle it but I want to get that resolved before moving forward.

Reply:
Suggested changes made.

Looks like the unit tests for this code were always prone to random race conditions.  To address the issue I added a WaitForFiles function to simplify the process of waiting for multiple files to exist (thanks Drew!), and also took out t.Parallel() to make that test run on its own.